.storybook/pages/CadStep1/CadStep1.tsx

```diff
@@ -0,0 +1,188 @@
+              fill="#383838"
+            />
+            <path
+              d="M118.46 233.15C121.62 234.95 112.77 241.24 101.09 247.38C99.3697 248.28 97.9397 245.77 99.5997 244.77C107.78 239.85 109.73 239.46 117.07 233.33C117.46 233.01 118.02 232.9 118.46 233.15V233.15Z"
+              fill="#C0C4C8"
+            />
+            <path
+              d="M317.19 102.89C320.35 104.69 311.5 110.98 299.82 117.12C298.1 118.02 296.67 115.51 298.33 114.51C306.51 109.59 308.46 109.2 315.8 103.07C316.19 102.75 316.75 102.64 317.19 102.89V102.89Z"
+              fill="#C0C4C8"
+            />
+            <path
+              d="M266.62 264.98C268.42 261.83 274.74 270.66 280.91 282.32C281.82 284.04 279.3 285.47 278.29 283.8C273.35 275.63 272.96 273.69 266.8 266.36C266.48 265.97 266.37 265.42 266.62 264.98Z"
+              fill="#C0C4C8"
+            />
+            <path
+              d="M135.8 66.6102C137.6 63.4502 143.92 72.2902 150.08 83.9502C150.99 85.6702 148.47 87.1002 147.46 85.4402C142.52 77.2702 142.13 75.3302 135.97 68.0002C135.65 67.6102 135.54 67.0602 135.79 66.6202L135.8 66.6102Z"
+              fill="#C0C4C8"
+            />
+          </svg>
+        </LayoutSection>
+        <LayoutSection region="main">
+          <Heading as="h2" className="u-margin-bottom-xl" size="h3">
+            Tell us how often your course meets.
+          </Heading>
+          <Text className="u-margin-bottom-xl" size="md">
+            We’ll adjust project plans based on your schedule. This way, you can
+            get a more accurate estimate of how long the course will actually
+            take.
+          </Text>
+          <Text className="u-margin-bottom-xl" size="md">
+            If you teach multiple sections of this course, you can enter the
+            average amounts for each question below.
+          </Text>
+          <TextField
```

Comment:
To me, this feels like the `Heading` component would live above the `TextField` and be the "How many times per week do you have class?" And then the text field label would be "Times per week" to read out properly to a user with Voiceover. Times per week doesn't seem like a field not and that that would be reserved for the error message.

That way, the screen reader would read: "Times per week: 7." What would get weird with the error message is that the "times per week" would get wiped out when there is an error too which we don't want.

Reply:
With Text moves the heading component above and resolves this issue